data/benefits/aide-au-bafa-pour-une-session-de-formation-générale.yml

```
@@ -0,0 +1,30 @@
+label: Aide au Bafa pour une session de formation générale (CAF du Val de Marne)
+institution: caf
+description: "L'aide locale du Val-de-Marne pour une session de formation
+  générale vient en complément de l'aide nationale. "
```

Il est possible d'ajouter des balises de lien dans la description comme ceci :
```suggestion
  générale vient en complément de <a target="_blank" rel="noopener" href="https://www.caf.fr/allocataires/vies-de-famille/jeune-ou-etudiant/vie-active/comment-financer-son-bafa">l'aide nationale</a>. "
```

---

je reformulerais le tout comme ceci pour donner un peu pus :
```
L'aide à la formation du Bafa est une aide locale mise à disposition par la caf du Val de Marne pour vous aider à financer votre formation en complément de <a target="_blank" rel="noopener" href="https://www.caf.fr/allocataires/vies-de-famille/jeune-ou-etudiant/vie-active/comment-financer-son-bafa">l'aide nationale apportée par la caf</a>.
```